thunder/core/proxies.py

```diff
@@ -129,9 +129,20 @@ def __init__(self, name: str | None = None, *, prefix: None | str = None, histor
     def name(self) -> str:
         return self._name
 
+    def replace(self, **changes):
+        r"""returns a copy replacing \**changes. Note that the copy will use the current tracectx"""
```

Comment:
I'm afraid I'm not quite following what this docstring would mean, could you update this?

Specifically "`\**changes" looks confusing to me

Reply:
how would you format this? (I found this as a way to write kwargs in docstrings and keep it proper rst for the doc generator.)